packages/cli/templates/default-app/packages/backend/src/index.ts.hbs

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Hi!
+ *
+ * Note that this is an EXAMPLE Backstage backend. Please check the README.
+ *
+ * Happy hacking!
+ */
+
+import {
+  createServiceBuilder,
+  getRootLogger,
+  useHotMemoize,
+} from '@backstage/backend-common';
+import { ConfigReader, AppConfig } from '@backstage/config';
+import { loadConfig } from '@backstage/config-loader';
+{{#if dbTypePG}}
+import knex, { PgConnectionConfig } from 'knex';
+{{/if}}
+{{#if dbTypeSqlite}}
+import knex from 'knex';
+{{/if}}
+import auth from './plugins/auth';
+import catalog from './plugins/catalog';
+import identity from './plugins/identity';
+import scaffolder from './plugins/scaffolder';
+import proxy from './plugins/proxy';
+import techdocs from './plugins/techdocs';
+import { PluginEnvironment } from './types';
+
+function makeCreateEnv(loadedConfigs: AppConfig[]) {
+  const config = ConfigReader.fromConfigs(loadedConfigs);
+
+  return (plugin: string): PluginEnvironment => {
+    const logger = getRootLogger().child({ type: 'plugin', plugin });
+
+    {{#if dbTypePG}}
+    const knexConfig = {
+      client: 'pg',
+      useNullAsDefault: true,
+      connection: {
+        port: process.env.POSTGRES_PORT,
+        host: process.env.POSTGRES_HOST,
+        user: process.env.POSTGRES_USER,
+        password: process.env.POSTGRES_PASSWORD,
```

Comment:
Should definitely move all of this to common to keep the template slim and upgradeable

Reply:
Other PR?🙃